src/utils/texture-selector-utils.ts

```diff
@@ -1,3 +1,5 @@
+// TODO Need to separate multiple export functions from file and split logic for better testing
```

Comment:
The file is just 78 rows. What do you want to split up here?

Reply:
Some export function are used inside this file. It is difficult to stub them from each other. It will be good to separate these functions to separate files.

Reply:
The file is unit. You don't need to stub functions inside the file.

Reply:
You can stub as well:
https://jestjs.io/docs/mock-functions#mocking-partials